Lib/concurrent/futures/process.py

```diff
@@ -239,6 +239,10 @@ def _process_worker(call_queue, result_queue, initializer, initargs):
         # Liberate the resource as soon as possible, to avoid holding onto
         # open files or shared memory that is not needed anymore
         del call_item
+        try:
```

Comment:
You just need to delete the result in the `else` clause of the previous `try` block unless I am missing something.